packages/next/build/webpack/plugins/next-drop-client-page-plugin.ts

```diff
@@ -1,29 +1,78 @@
 import { Compiler, Plugin } from 'webpack'
-import { extname } from 'path'
+import { STRING_LITERAL_DROP_BUNDLE } from '../../../next-server/lib/constants'
+
+const PLUGIN_NAME = 'DropAmpFirstPagesPlugin'
+
+// Recursively look up the issuer till it ends up at the root
+function findEntryModule(issuer: any): any {
+  if (issuer.issuer) {
```

Comment:
This might trip you up if this module is included in multiple entrypoints. `issuer` will only give you the first requester. `module.reasons` give you all, but it's more difficult to traverse.